api/v1alpha1/akodeploymentconfig_webhook.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	"fmt"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// log is for logging in this package.
+var akoDeploymentConfigLog = logf.Log.WithName("akodeploymentconfig-resource")
```

Comment:
Is it possible to use `go.uber.org/zap` package for logger? Just want to align with other places we use in the repo.

Reply:
This was generated by kubebuilder, I can check to see if we can use the zap package for logger